index.css

```diff
@@ -0,0 +1,167 @@
+*,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+li {
+  margin: 0px;
+  padding: 0px;
+}
+body {
+  font-family: Arial, sans-serif;
+  background: #f5f5f5;
+}
+
+#todo-app {
+  max-width: 913px;
```

Comment:
Use `rem` and `em` instead of `px`: https://elementor.com/help/whats-the-difference-between-px-em-rem-vw-and-vh/#:~:text=EM%20is%20relative%20to%20the,parent%20size%20is%2C%20use%20REM. 

-5

Reply:
Yes I will take care of it.